src/components/ChatBotInput/ChatBotInput.tsx

```diff
@@ -103,12 +106,27 @@ const ChatBotInput = ({ buttons }: { buttons: JSX.Element[] }) => {
 		setIsFocused(false);
 	};
 
+	/**
+	 * Handles composition start event on chat input.
+	 */
+	const handleCompositionStart = () => {
+		setIsComposing(true);
+	};
+
+	/**
+	 * Handles composition end event on chat input.
+	 */
+	const handleCompositionEnd = () => {
+		setIsComposing(false);
+	};
+
 	/**
 	 * Handles keyboard events and proceeds to submit user input if enter button is pressed.
 	 * 
 	 * @param event keyboard event
 	 */ 
 	const handleKeyDown = async (event: KeyboardEvent<HTMLTextAreaElement | HTMLInputElement>) => {
+		if (isComposing) return;
```

Comment:
Let's avoid one-liners and use egyptian braces instead.

Reply:
Fixed.
[chore: use egyptian braces](https://github.com/tjtanjin/react-chatbotify/pull/112/commits/1bcc299688bdb0f2ba908f00e475a653a10cc217)